static/index.json

```diff
@@ -11,7 +11,7 @@
     },
     {
         "uri": "/noaa",
-        "content": "---\ntitle: \"Noaa\"\nslug: \"noaa\"\nsubtitle: \"Stock Market Data\" \ndraft: false\nimage: img/data-playground/finnhub.png\ngithub_link: \"github.com\"\ndescription: FinnHub provides real-time stock market data. Note that the stock market is closed during certain times of days and days of the week.\nsummary: Connect to stock market data and start experimenting with financial models and apps.\nlicense: Free\nlicense_warning: Please review terms and conditions. Access to data sources can change.\nproducer_name: FinnHub\nproducer_link: https://finnhub.io/\nisapikey_required: true\napi_type: REST, Websocket\nsdks: Go\nlimits: N/A\ndata: Stock prices, company profiles, company & market news\nisaccountrequired: true\nweight: 2\n---\n\nAdd data products showcase \n\n",
+        "content": "---\ntitle: \"Noaa\"\nslug: \"noaa\"\nsubtitle: \"Weather Data\" \ndraft: false\nimage: img/data-playground/noaa.png\ngithub_link: \"https://github.com/rotationalio/data-playground/tree/main/weather\"\ndescription: The National Oceanic and Atmospheric Administration's (NOAA's) National Weather Service offers critical real-time data. Get forecasts, alerts, observations, and other weather data.\nsummary: The National Weather Service (NWS) API allows access to critical forecasts, alerts, and observations, along with other weather data.\nlicense: Free\nlicense_warning: Please review terms and conditions. Access to data sources can change.\nproducer_name: NOAA\nproducer_link: https://api.weather.gov\nisapikey_required: false\napi_type: REST, JSON-LD, GeoJSON, OXML\nsdks: N/A\nlimits: The rate limit is not public information, but allows a generous amount for typical use. If the rate limit is exceeded, the request will return with an error, and may be retried after the limit clears (typically within 5 seconds). Proxies are more likely to reach the limit, whereas requests directly from clients are not likely.\ndata: Stock prices, company profiles, company & market news\nisaccountrequired: false\nweight: 2\n---\n\nAdd data products showcase \n\n",
```

Comment:
The `title` isn't in all caps here like it is in the `/noaa.md` file. Also, it looks like the `data` still has some Finnhub info. Could you run `yarn index` again to make sure that gets updated?